### DISCUSSION TITLE
Rotation angle

### DISCUSSION THREAD

**Discussion by Ken-Kuroki** [UTokyo-FieldPhenomics-Lab/EasyIDP#100] [Q&A] [answered]
Hello!

I wonder how EasyIDP deals with image rotation when backward projecting. I mean, if a shape file has a rectangle, which edge will be the top (i.e. defined by the 0-th and 1-st entries of the output image coordinate array) of the resulting crop region? Does the edge between 9 to 12 o'clock become the top?

**Comment by HowcanoeWang** [comment, marked as answer]
The easyidp doesn't rotate and resize the rectange to get the final position. 

It can only calcualte a 3D point (x, y, z) to 2D point (x0, y0) on the image

And a rectange has 4 vectex, repeat previous step for each vertex to get the final result.

**Reply by Ken-Kuroki** [reply]
Got it. Thank you!
